client/modules/datafiles/src/publications/PublicationSearchToolbar/PublicationSearchToolbar.tsx

```diff
@@ -1,13 +1,59 @@
-import { Button, Form, Input, Select } from 'antd';
-import React from 'react';
+import { Button, Form, Input, Collapse } from 'antd';
+import { CloseOutlined } from '@ant-design/icons';
+import React, { useState, useEffect, useRef } from 'react';
 import { useSearchParams } from 'react-router-dom';
-import * as dropdownOptions from '../../projects/forms/ProjectFormDropdowns';
+
+const { Panel } = Collapse;
+
 export const PublicationSearchToolbar: React.FC = () => {
   const [searchParams, setSearchParams] = useSearchParams();
+  const [recentSearches, setRecentSearches] = useState<string[]>([]);
+  const [showTips, setShowTips] = useState(false);
+  const searchBarRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
```

Comment:
I think this would be a good place to use the [useSyncExternalStore](https://lo-victoria.com/a-look-at-react-hooks-usesyncexternalstore#heading-minimalistic-example) hook instead of useState/useEffect to grab the value from localStorage. That hook has the advantage of keeping the recent searches synced across however many windows/browser tabs the user has open.